app/models/user.rb

```diff
@@ -273,6 +273,10 @@ def power_of_attorney
     EVSS::CommonService.get_current_info[:poa]
   end
 
+  def flipper_id
```

Comment:
I don't see this used anywhere.  Is this used internally by Flipper?

Reply:
yeah, I'll ad a comment. It can be used to enable a feature for individual users, 
<img width="514" alt="Screen Shot 2019-08-07 at 11 43 25 AM" src="https://user-images.githubusercontent.com/19188/62637047-b6d80c80-b908-11e9-815c-a3877305c654.png">

but it's also used in the algorithm for enabling a feature for x% of users